test/producer-consumer-spec.js

```diff
@@ -141,4 +141,17 @@ describe('producer/consumer', function () {
       });
     });
   });
+
+  describe('undefined queue name', () => {
+    before(() => {
+      bunnymq.connection._config.consumerSuffix = undefined;
+    });
+
+    it('should receive message', () => {
+      bunnymq.consume('queue-name-undefined-suffix', (message) => {
+        assert.equal(message.msg, 'test for undefined queue suffix');
+      })
```

Comment:
my suggestion `).then`
I think It would be better if we will use same approach everywhere As you think @dial-once/developers ?